reports-rest/src/main/java/org/jboss/da/rest/listings/Artifacts.java

```diff
@@ -112,8 +112,10 @@ public Response fillFromGitBom(
                 response.setMessage("Error while analysing pom file");
                 return Response.status(Status.BAD_REQUEST).entity(response).build();
         }
-        return Response.status(Status.INTERNAL_SERVER_ERROR)
-                .entity(new ErrorMessage("There was some internal error")).build();
+        return Response
+                .status(Status.INTERNAL_SERVER_ERROR)
+                .entity(new ErrorMessage(ErrorMessage.eType.INTERNAL_ERR,
+                        "There was some internal error")).build();
```

Comment:
Make this a default case in the switch and return unexpected server error. See ``addWhiteArtifact`` below.